### PR TITLE
feat: Migration page integration

### PR DESCRIPTION
**Pull request by ChefMomota**

### CODE REVIEW

File: src/views/Farms/components/MigrationSticky.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      if (warningBannerHeight) {
+        totalHeight = navHeight + (warningBannerHeight as any).offsetHeight
+      }
+      setStickyPosition(totalHeight)
+    }
+    lastScroll = currentScroll
+  }
+
+  return (
+    <Container style={{ top: `${stickPosition}px` }}>
```
Comment (0xjojoex) [inline src/views/Farms/components/MigrationSticky.tsx:96]: We can just use position sticky here, top can be 0 or 126
A quick way to do it is add a class name `menu-show`on body https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/uikit/src/widgets/Menu/Menu.tsx#L104-L109

File: src/state/farms/getFarmsPrices.ts [hunk 2]
```diff
@@ -86,7 +86,8 @@ const getFarmQuoteTokenPrice = (
 }
 
 const getFarmsPrices = (farms: SerializedFarm[]) => {
-  const bnbBusdFarm = farms.find((farm) => farm.token.symbol === 'BUSD' && farm.quoteToken.symbol === 'WBNB')
+  // const bnbBusdFarm = farms.find((farm) => farm.token.symbol === 'BUSD' && farm.quoteToken.symbol === 'WBNB')
+  const bnbBusdFarm = farms.find((farm) => farm.pid === 2)
```
Comment (0xjojoex) [inline src/state/farms/getFarmsPrices.ts]: revert this before merge

File: src/state/farms/hooks.ts [hunk 3]
```diff
@@ -45,7 +45,8 @@ export const usePollCoreFarmData = () => {
   const dispatch = useAppDispatch()
 
   useFastRefreshEffect(() => {
-    dispatch(fetchFarmsPublicDataAsync(coreFarmPIDs))
+    // dispatch(fetchFarmsPublicDataAsync(coreFarmPIDs))
+    dispatch(fetchFarmsPublicDataAsync([1, 2]))
```
Comment (0xjojoex) [inline src/state/farms/hooks.ts]: revert this before merge

File: src/state/farms/hooks.ts [hunk 4]
```diff
@@ -87,5 +88,15 @@ export const useLpTokenPrice = (symbol: string) => {
  * @@deprecated use the BUSD hook in /hooks
  */
 export const usePriceCakeBusd = (): BigNumber => {
-  return useSelector(priceCakeFromPidSelector)
+  // return useSelector(priceCakeFromPidSelector)
+
+  const cakeBnbFarm = useFarmFromPid(1)
+
+  const cakePriceBusdAsString = cakeBnbFarm.tokenPriceBusd
```
Comment (0xjojoex) [inline src/state/farms/hooks.ts]: revert this before merge

File: src/state/farmsV1/getFarmsPrices.ts [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  if (quoteTokenFarm.quoteToken.symbol === 'BUSD') {
+    return quoteTokenFarm.tokenPriceVsQuote ? new BigNumber(quoteTokenFarm.tokenPriceVsQuote) : BIG_ZERO
+  }
+
+  return BIG_ZERO
+}
+
+const getFarmsPrices = (farms: SerializedFarm[]) => {
+  // const bnbBusdFarm = farms.find((farm) => farm.v1pid === 252)
+  const bnbBusdFarm = farms.find((farm) => farm.v1pid === 2)
```
Comment (0xjojoex) [inline src/state/farmsV1/getFarmsPrices.ts]: revert this before merge

File: src/state/farmsV1/hooks.ts [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+ * Fetches the "core" farm data used globally
+ * 251 = CAKE-BNB LP
+ * 252 = BUSD-BNB LP
+ */
+export const usePollCoreFarmData = () => {
+  const dispatch = useAppDispatch()
+
+  useFastRefreshEffect(() => {
+    // dispatch(fetchFarmsPublicDataAsync([251, 252]))
+    dispatch(fetchFarmsPublicDataAsync([1, 2]))
```
Comment (0xjojoex) [inline src/state/farmsV1/hooks.ts]: revert this before merge

File: src/state/farmsV1/hooks.ts [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+  return lpTokenPrice
+}
+
+/**
+ * @@deprecated use the BUSD hook in /hooks
+ */
+export const usePriceCakeBusd = (): BigNumber => {
+  // const cakeBnbFarm = useFarmFromPid(251)
+  const cakeBnbFarm = useFarmFromPid(1)
```
Comment (0xjojoex) [inline src/state/farmsV1/hooks.ts]: revert this before merge

File: src/config/constants/contracts.ts [hunk 8]
```diff
@@ -52,7 +58,8 @@ export default {
     97: '0x24ec6962dbe874F6B67B5C50857565667fA0854F',
   },
   cakeVault: {
-    56: '0xa80240Eb5d7E05d3F250cF000eEc0891d00b51CC',
+    // 56: '0xa80240Eb5d7E05d3F250cF000eEc0891d00b51CC',
+    56: '0xd72b4Ac307a9E0c5Ca85878b1ABF980D4A5eE490',
```
Comment (ChefMomota) [inline src/config/constants/contracts.ts]: revert this before merge

File: src/state/farms/hooks.ts [hunk 9]
```diff
@@ -137,7 +138,8 @@ export const useLpTokenPrice = (symbol: string) => {
  * @@deprecated use the BUSD hook in /hooks
  */
 export const usePriceCakeBusd = (): BigNumber => {
-  const cakeBnbFarm = useFarmFromPid(251)
```
Comment (ChefMomota) [inline src/state/farms/hooks.ts]: revert this before merge

File: src/state/farms/getFarmsPrices.ts [hunk 10]
```diff
@@ -86,7 +86,8 @@ const getFarmQuoteTokenPrice = (
 }
 
 const getFarmsPrices = (farms: SerializedFarm[]) => {
-  const bnbBusdFarm = farms.find((farm) => farm.pid === 252)
```
Comment (ChefMomota) [inline src/state/farms/getFarmsPrices.ts]: revert this before merge

File: src/config/constants/tokens.ts [hunk 11]
```diff
@@ -212,7 +213,8 @@ export const mainnetTokens = defineTokens({
   ),
   syrup: new Token(
     MAINNET,
-    '0x009cF7bC57584b7998236eff51b98A168DceA9B0',
+    // '0x009cF7bC57584b7998236eff51b98A168DceA9B0',
```
Comment (ChefMomota) [inline src/config/constants/tokens.ts]: revert this before merge

File: src/config/constants/tokens.ts [hunk 12]
```diff
@@ -24,7 +24,8 @@ export const mainnetTokens = defineTokens({
   bnb: new Token(MAINNET, '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c', 18, 'BNB', 'BNB', 'https://www.binance.com/'),
   cake: new Token(
     MAINNET,
-    '0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82',
+    // '0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82',
+    '0xe5e76f3BA7fE02d7FCf4FffFedDEF6cc873Fa135',
```
Comment (ChefMomota) [inline src/config/constants/tokens.ts]: revert this before merge

File: src/config/constants/pools.tsx [hunk 13]
```diff
@@ -36,7 +36,8 @@ const pools: SerializedPoolConfig[] = [
     earningToken: serializedTokens.cake,
     contractAddress: {
       97: '0x1d32c2945C8FDCBc7156c553B7cEa4325a17f4f9',
-      56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+      // 56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+      56: '0x2782030FaaEc9F6DaC96cDA9c1dF3125A015078D',
```
Comment (ChefMomota) [inline src/config/constants/pools.tsx]: change new contract address before merge

File: src/config/constants/contracts.ts [hunk 14]
```diff
@@ -1,11 +1,13 @@
 export default {
   masterChef: {
-    97: '0x1d32c2945C8FDCBc7156c553B7cEa4325a17f4f9',
-    56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+    97: '',
+    // 56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+    56: '0x327E6e33E4b31CCBd6edb31c788aBA3b6fdb5F1F',
```
Comment (ChefMomota) [inline src/config/constants/contracts.ts]: change new contract address before merge

File: src/config/constants/contracts.ts [hunk 15]
```diff
@@ -1,11 +1,13 @@
...
   },
   masterChefV1: {
     97: '0x1d32c2945C8FDCBc7156c553B7cEa4325a17f4f9',
-    56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+    // 56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+    56: '0x2782030FaaEc9F6DaC96cDA9c1dF3125A015078D',
```
Comment (ChefMomota) [inline src/config/constants/contracts.ts]: revert this before merge

File: src/views/Migration/hook/V1/Pool/useFetchUserPools.ts [hunk 16]
```diff
@@ -25,7 +25,8 @@ export const useFetchUserPools = (account) => {
       earningToken: serializedTokens.cake,
       contractAddress: {
         97: '0x1d32c2945C8FDCBc7156c553B7cEa4325a17f4f9',
-        56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+        // 56: '0x73feaa1eE314F8c655E354234017bE2193C9E24E',
+        56: '0x2782030FaaEc9F6DaC96cDA9c1dF3125A015078D',
```
Comment (ChefMomota) [inline src/views/Migration/hook/V1/Pool/useFetchUserPools.ts]: revert this before merge

File: .env.production [hunk 17]
```diff
@@ -11,7 +11,7 @@ NEXT_PUBLIC_NODE_2 = "https://bsc-dataseed1.defibit.io"
 NEXT_PUBLIC_NODE_3 = "https://bsc-dataseed.binance.org"
 
 # Google Cloud Infrastructure Endpoint - Global
-NEXT_PUBLIC_NODE_PRODUCTION = "https://nodes.pancakeswap.com"
```
Comment (ChefMomota) [inline .env.production:14]: revert this before merge

File: .env.production [hunk 18]
```diff
@@ -11,7 +11,7 @@ NEXT_PUBLIC_NODE_2 = "https://bsc-dataseed1.defibit.io"
...
+NEXT_PUBLIC_NODE_PRODUCTION = "https://bsc-dataseed.binance.org"
```
Comment (ChefMomota) [inline .env.production]: revert this before merge

File: src/state/farms/hooks.ts [hunk 19]
```diff
@@ -87,5 +88,15 @@ export const useLpTokenPrice = (symbol: string) => {
  * @@deprecated use the BUSD hook in /hooks
  */
 export const usePriceCakeBusd = (): BigNumber => {
-  return useSelector(priceCakeFromPidSelector)
+  // return useSelector(priceCakeFromPidSelector)
```
Comment (chefilip) [inline src/state/farms/hooks.ts:91]: I think we just need to update `selectCakeFarm` into `0` ?

Comment (chefilip) [inline src/state/farms/hooks.ts]: Why do we need to use price in farm instead syrup ?

**Pull request closed without merging**